src/sections/SideBar/components/ProfilePic/SideBarProfile.js

```diff
@@ -0,0 +1,49 @@
+      >
+        C
```

Comment:
typo error?

Reply:
It's a feature of Material UI. If the Avatar element cannot load the image, it displays the text between its tags instead.

Reply:
Then put my name just in case

Reply:
Got it, sir!